internal/releaseversion/releaseversion.go

```diff
@@ -88,3 +137,12 @@ func included(releases []*semver.Version, release semver.Version) bool {
 func isAlphaRelease(release string) bool {
 	return strings.Contains(release, "alpha")
 }
+
+func isIgnoredRelease(releaseCR *v1alpha1.Release) bool {
+	ignoreValue, isIgnoreAnnotationSet := releaseCR.Annotations[ignoreReleaseAnnotation]
```

Comment:
I'm not sure if it even can happen, but should we check if `Annotations` is not nil?

Reply:
Not sure, good point :+1: :fist_oncoming:

Reply:
It does not break for `Annotations==nil`, it just returns `isIgnoreAnnotationSet` as `false`, example https://play.golang.org/p/2n40nlvAcby

Reply:
We have too many `null` traumas from other languages :smile: